libs/vkd3d/utils.c

```diff
@@ -913,6 +916,139 @@ bool is_valid_resource_state(D3D12_RESOURCE_STATES state)
     return true;
 }
 
+bool is_valid_format(DXGI_FORMAT dxgi_format)
+{
+    switch(dxgi_format)
+    {
+    case DXGI_FORMAT_UNKNOWN:
```

Comment:
Can this be replaced with a range check instead of a humongous switch? At a glance it looks like all enums from 0 to the maximum value are considered "valid".